roles/etcd_ca/tasks/main.yml

```diff
@@ -2,6 +2,8 @@
 - name: Install openssl
   action: "{{ ansible_pkg_mgr }} name=openssl state=present"
   when: not openshift.common.is_atomic | bool
+  delegate_to: "{{ openshift_etcd_ca_host }}"
```

Comment:
If this is going to be a generic etcd role (and I think it can and should be), then we need to avoid using openshift_* for variables, and use an openshift_etcd_facts role to translate values from openshift_facts into etcd_* variables.